modules/apps/calendar/calendar-test-util/src/main/java/com/liferay/calendar/test/util/CalendarBookingUpgradeProcessTestUtil.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.calendar.test.util;
+
+import com.liferay.calendar.model.CalendarBooking;
+import com.liferay.portal.kernel.model.Group;
+import com.liferay.portal.kernel.model.User;
+import com.liferay.portal.kernel.service.ServiceContext;
+import com.liferay.portal.kernel.service.UserLocalService;
+import com.liferay.portal.kernel.test.util.GroupTestUtil;
+import com.liferay.portal.kernel.test.util.ServiceContextTestUtil;
+import com.liferay.portal.kernel.util.CalendarFactoryUtil;
+
+import java.util.Calendar;
+
+/**
+ * @author Richard Jeremias
+ */
+public class CalendarBookingUpgradeProcessTestUtil {
+
+	public static CalendarBooking createCalendarBooking(
+			User user, UserLocalService userLocalService)
```

Comment:
I'm not sure if it's a ok practice to send a localService object as a param. It was necessary because I intend to update the current user.